index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      body {
+        background-color: pink;
+      }
+      
+      @media screen and (min-width: 480px) {
+        body {
+          background-color: lightgreen;
+        }
+      }
+      </style>
+  </head>
+  
+  <body>
+
+    <nav id="navbar">
+      <a href="#welcome-section" id="profile-link" target="_blank">projects</a>
+     </nav>
+     <section id="welcome-section" name="projects">
+      <h1>Test</h1>
+      <h1>hello</h1>
+     </section>
+     <br>
+     <!-- <hr> -->
+
+     <section id="projects" >
+      <h2 class="project-tile"></h2>
+      <a href="https://www.google.com/webhp?hl=en&sa=X&ved=0ahUKEwjyl_2lvZL7AhW2i_0HHcG7BZ8QPAgI"></a>
+    </>
```

Comment:
please, always remember that the semantic tags always have a closing tag like `<section>` here you need to close it with `</section>` in line 35